riptide-stream/src/main/java/org/zalando/riptide/stream/StreamConverter.java

```diff
@@ -32,15 +31,26 @@
             Collections.unmodifiableList(Arrays.asList(APPLICATION_JSON_SEQ, APPLICATION_X_JSON_STREAM));
 
     private final ObjectMapper mapper;
-    private final List<MediaType> medias;
+    private final List<MediaType> supportedMediaTypes;
 
+    /**
+     * @deprecated Use {@link Streams#streamConverter(ObjectMapper)}
+     */
+    @Deprecated
```

Comment:
There shouldn't be a possibility to not pass in a mapper. Clients should always pass one.